skeleton/stack.yaml

```diff
@@ -0,0 +1,36 @@
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/d673b92f2a2d9354d102514d0a0fa74f8248b14a/snapshot.yaml
```

Comment:
Is this the recommended resolver to use? Can/should we use LTS resolvers?

Reply:
We have CI on the prelude one now, so if you point to latest prelude, you can guarantee iohk-nix/haskell.nix have support for that LTS already.